==== triton/BUILD#44 - /google/src/cloud/csigg/mlir_transform_utils/triton/BUILD ====
# action=edit type=text
--- triton/BUILD	2024-03-22 08:02:38.000000000 -0700
+++ triton/BUILD	2024-03-27 01:34:43.000000000 -0700
@@ -620,6 +620,7 @@
         "@llvm-project//mlir:FunctionInterfaces",
         "@llvm-project//mlir:GPUDialect",
         "@llvm-project//mlir:IR",
+        "@llvm-project//mlir:InliningUtils",
         "@llvm-project//mlir:LLVMDialect",
         "@llvm-project//mlir:MathDialect",
         "@llvm-project//mlir:SCFDialect",
@@ -628,6 +629,7 @@
         # The following is added to make Utility compile
         ":TritonTools",
         "@llvm-project//mlir:LLVMCommonConversion",
+        "@llvm-project//mlir:TransformUtils",
         "@llvm-project//mlir:Transforms",
     ],
 )
@@ -646,6 +648,7 @@
         "@llvm-project//mlir:IR",
         "@llvm-project//mlir:Pass",
         "@llvm-project//mlir:Support",
+        "@llvm-project//mlir:TransformUtils",
         "@llvm-project//mlir:Transforms",
     ],
     alwayslink = True,  # TritonDialect uses getCanonicalizationPatterns().
@@ -729,6 +732,7 @@
         "@llvm-project//mlir:LLVMDialect",
         "@llvm-project//mlir:NVVMDialect",
         "@llvm-project//mlir:Pass",
+        "@llvm-project//mlir:TransformUtils",
         "@llvm-project//mlir:Transforms",
     ],
 )
@@ -780,6 +784,7 @@
         "@llvm-project//mlir:IR",
         "@llvm-project//mlir:IndexDialect",
         "@llvm-project//mlir:Pass",
+        "@llvm-project//mlir:TransformUtils",
         "@llvm-project//mlir:Transforms",
     ],
 )
==== triton/test/BUILD#18 - /google/src/cloud/csigg/mlir_transform_utils/triton/test/BUILD ====
# action=edit type=text
--- triton/test/BUILD	2024-03-11 11:42:57.000000000 -0700
+++ triton/test/BUILD	2024-03-27 01:32:04.000000000 -0700
@@ -53,6 +53,7 @@
         "@llvm-project//mlir:IR",
         "@llvm-project//mlir:Pass",
         "@llvm-project//mlir:SCFToControlFlow",
+        "@llvm-project//mlir:TransformUtils",
         "@llvm-project//mlir:Transforms",
         "//:TritonAnalysis",
         "//:TritonDialects",
==== triton/third_party/nvidia/BUILD#3 - /google/src/cloud/csigg/mlir_transform_utils/triton/third_party/nvidia/BUILD ====
# action=edit type=text
--- triton/third_party/nvidia/BUILD	2024-03-11 11:42:57.000000000 -0700
+++ triton/third_party/nvidia/BUILD	2024-03-27 01:32:46.000000000 -0700
@@ -66,6 +66,7 @@
         "@llvm-project//mlir:NVVMDialect",
         "@llvm-project//mlir:Pass",
         "@llvm-project//mlir:Support",
+        "@llvm-project//mlir:TransformUtils",
         "@llvm-project//mlir:Transforms",
         "//:TritonDialects",
     ],
@@ -113,6 +114,7 @@
         "@llvm-project//mlir:NVVMDialect",
         "@llvm-project//mlir:Pass",
         "@llvm-project//mlir:SCFToControlFlow",
+        "@llvm-project//mlir:TransformUtils",
         "@llvm-project//mlir:Transforms",
         "//:TritonAnalysis",
         "//:TritonDialects",
